tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 minversion = 2.0
-envlist = py36,py27,pep8,docs
```

Comment:
Is there a reason `docs` is removed from the envlist?

Reply:
Oops, no. My bad. I will be putting it back shortly.